contrib/pylightning/lightning/lightning.py

```diff
@@ -161,7 +161,8 @@ def invoice(self, msatoshi, label, description, expiry=None, fallbacks=None, pre
             "description": description,
             "expiry": expiry,
             "fallbacks": fallbacks,
-            "preimage": preimage
+            "preimage": preimage,
+            "maxroutes": maxroutes
```

Comment:
`maxroutes` isn't really the most intuitive name. Can we make it something like `privchannels`?

Reply:
`maxprivchannels`?